### PR TITLE
Support for Node 15

### LINKED ISSUE #12: Incompatibility with node.js 15.9.0
Hello, I'm trying to do `yarn install` on my serverless project. the `serverless-bundle` has this package as a dependency but every time it will hit me with the following error:

`error concat-text-webpack-plugin@0.1.9: The engine "node" is incompatible with this module. Expected version ">=8 <=14". Got "15.9.0"
error Found incompatible module.`

Is there any chance you can allow compatibility with this version of node, or maybe the error is on their part? Thank you.

### PR DESCRIPTION
**Pull request by manuelaguirre**

### CODE REVIEW

**Comment by manuelaguirre** [comment]
I extended the previous PR with what you asked its author

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
   test:
     strategy:
       matrix:
-        node: [12, 14]
+        node: [12, 15]
```
Comment (zidizei) [inline .github/workflows/publish.yml:11]: Please leave in the test matrix value for Node 14. Same goes for the matrices in `test.yml`.

```suggestion
        node: [12, 14, 15]
```

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "concat-text-webpack-plugin",
-  "version": "0.1.9",
+  "version": "0.1.10",
```
Comment (zidizei) [inline package.json:3]: We don't need to specify the version manually in the PR. This value will be updated once we prepare this for a release.

**Review by zidizei — CHANGES_REQUESTED** [review]
Thanks for picking this up! I've added some minor change suggestions to your PR. Once that's been taken care of, we can get this thing on the road! 😃

**Comment by zidizei** [comment]
Superseded by #16

**Pull request closed without merging**